spelling fixes and a better warning for not using DXVK

dribbleondo opened the pull request:
There were a load of spelling errors, and weird "heres whats going wrong, heres how proton fixed it". Generally, if people want to look at an FAQ for answers, you should probably assure them that a later version of proton has fixed their issue first, then the troubleshooting steps next.

dribbleondo committed:
spelling fixes and a better warning for not using DXVK

There were a load of spelling errors, and weird "heres whats going wrong, heres how proton fixed it". Generally, if people want to look at an FAQ for answers, you should probably assure them that a later version of proton has fixed their issue first, then the troubleshooting steps next.

dribbleondo committed:
minor spelling error

sorry

Alexander88207 commented on README.md:
```diff
@@ -41,61 +41,62 @@ With this FAQ, we want to cover the most important questions related to the usag
+## Why do my games crash on startup, run very slowly or have rendering issues?
 
-#### Be sure that your system is up-to-date and that you are using the latest drivers available your graphics card. You will find detailed informations about how to update your graphic drivers in our [Wiki](https://github.com/NoXPhasma/protondb_faq/wiki/Graphics-driver-installation)
+#### Be sure that your system is up-to-date and that you are using the latest drivers available for your graphics card. Generally, for AMD and Intel, you should have be using a recent version of Mesa (19+) if you're on AMD or Intel, and Nvidia users should insteall the proprietary drivers from Nvidia's site:  You will find more detailed information about how to update your graphic drivers on our  ...
```
Its not recommended to install nvidia from nvidia's site.

dribbleondo replied:
Done.

dribbleondo committed:
as per request, changed Nvidia line

as per request, changed Nvidia line to not recommend installing from Nvidias website

dribbleondo commented:
Okay, I'll remove that.

dribbleondo committed:
removed general instructions for drivers

will add this to the wiki soon.

ghost commented:
LGTM, Thanks for your contribution!
